README.md

```diff
@@ -9,3 +9,7 @@ An App to display temporal and temperature graphical files for Isomemo.
 
 ### Release notes:
 - see `NEWS.md`
+
+## How to use this Package
+
+Refer to the vignette for a description of the usage of the MapR package.
```

Comment:
```suggestion
Refer to the vignette for a description of the usage of the MapR package.

----

### Notes for developers

When adding information to _help_ sites or the _vignette_ of this package, please update the static HTML documentation with `devtools::build_site()`, and push all changes.
```

Reply:
- [ ] We need to add the link to the HTML documentation to the readMe, e.g. just put the link of the HTML vignette under the word "vignette" above. (as soon it is available online.)

Reply:
I added this task to the new [Documentaion Issue](https://github.com/Pandora-IsoMemo/MapR/issues/18). I will start with the automation of the doc as soon as this branch is merged into the main branch.

Reply:
Sorry I have to reopen this, because it is not quite right, once we automate the documentation workflow. The workflow will always rebuild the documention, when there is change on the main branch. The workflow below is only for locally building the doc.

```suggestion
Refer to the vignette for a description of the usage of the MapR package.

----

### Notes for developers

When adding information to _help_ sites, _docstrings_ or the _vignette_ of this package, 
please update documentation locally as follows:

devtools::document() # or CTRL + SHIFT + D in RStudio
devtools::build_site()
```